audited.gemspec

```diff
@@ -18,23 +18,24 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(/^spec\//)
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'activerecord', '~> 4.0'
+  gem.add_dependency 'activerecord', '< 6.0', '>= 5.0.0'
   gem.add_dependency 'rails-observers', '~> 0.1.2'
 
-  gem.add_development_dependency "protected_attributes"
+  # gem.add_development_dependency "protected_attributes"
   gem.add_development_dependency 'appraisal', '~> 1.0.0'
-  gem.add_development_dependency 'rails', '~> 4.2.0'
-  gem.add_development_dependency 'rspec-rails', '~> 3.0'
+  gem.add_development_dependency 'rails', '< 6.0', '>= 5.0.0'
+  gem.add_development_dependency 'rspec-rails'
+  gem.add_development_dependency 'active_record_upsert'
```

Comment:
Not a fan of inter-gem dependencies that both extend ActiveRecord like this... but maybe necessary for now if we intend to keep `upsert` in Rails 5.

Rails 6 introduces `upsert` as a no-frills no-callbacks database statement and I think we'll eventually want to use that. So adding a dependency on the gem here might proliferate use of `upsert` that depends on similar callbacks, not just Audited, and it could make it harder to untangle later.